geode-core/src/main/java/org/apache/geode/internal/cache/CachePerfStats.java

```diff
@@ -442,7 +451,10 @@
             f.createIntCounter("retries",
                 "Number of times a concurrent destroy followed by a create has caused an entry operation to need to retry.",
                 "operations"),
-            f.createLongCounter("clears", clearsDesc, "operations"),
+            f.createLongCounter("regionClears", regionClearsDesc, "operations"),
```

Comment:
If the name of this statistic is changed, then `geode-docs/reference/statistics_list.html.md.erb` should be updated to reflect the new name. Also, it might be good to document the newly added statistics too.

Reply:
Agreed. I recall there was a documentation story for PR clear, but I'm not sure what happened to it. I'll hunt it down and if this documentation isn't covered by it I'll make sure to add it to this PR.